npm/webpack-dev-server/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "build": "tsc",
     "build-prod": "tsc",
-    "test": "tsc && mocha -r @packages/ts/register test/**/*.spec.ts --exit",
+    "test": "tsc && mocha -r @packages/ts/register test/**/*.spec.ts test/*.spec.ts --exit",
```

Comment:
Undo this?

Reply:
Actually this is necessary - otherwise `test/e2e.spec.ts` is not run. It was never running on CI, apparently.

I am not sure why the glob `test/**/*.spec.ts` does not capture `test/e2e.spec.ts`, but does capture `test/unit/makeWebpack.spec.ts`.